wordpress-mobile/WordPress-Android
# Issue/editor beta label

### Fix
Add ***Beta*** label to editor title when Aztec is enabled.  When the label is tapped, the ***What's New / Release Notes*** interface is shown.

### Test
0. Enable ***Beta*** editor.
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Notice ***Beta*** label to right of editor title.
4. Tap ***Beta*** label.
5. Notice ***What's New / Release Notes*** interface.
6. Tap back arrow.
7. Notice editor is shown.
